pkg/controller/podgc/gc_controller_test.go

```diff
@@ -117,33 +123,35 @@ func TestGCTerminated(t *testing.T) {
 	}
 
 	for i, test := range testCases {
-		client := fake.NewSimpleClientset(&v1.NodeList{Items: []v1.Node{*testutil.NewNode("node")}})
-		gcc, podInformer, _ := NewFromClient(client, test.threshold)
-		deletedPodNames := make([]string, 0)
-		var lock sync.Mutex
-		gcc.deletePod = func(_, name string) error {
-			lock.Lock()
-			defer lock.Unlock()
-			deletedPodNames = append(deletedPodNames, name)
-			return nil
-		}
+		t.Run(fmt.Sprintf("case: %v", i), func(t *testing.T) {
```

Comment:
as I mentioned above, instead of "case.." use the description/name field of the tst you will introduce

Reply:
Fixed.